src/PhpHooks/Command/ForbiddenCommand.php

```diff
@@ -53,4 +53,25 @@ public function run(InputInterface $input, OutputInterface $output)
             }
         }
     }
+
+    /**
+     * @param string $filePath
+     * @param array $configuration
+     * @return bool
+     */
+    protected function checkExcludeFile($filePath, array $configuration)
```

Comment:
Methodnames that starts with "check" are not meaningful. Please change the name in something similiar like "isExcluded" or "isExcludedFile".